.github/workflows/js_test.yml

```diff
@@ -0,0 +1,20 @@
+name: Javascript testing
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  eslint:
```

Comment:
and eslint isn't relevant on the testing job is it? Just used for the linting?

Reply:
Ugh, teach me to copy paste templates.